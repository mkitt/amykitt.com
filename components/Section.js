// @flow
import React from 'react'
import css, { media3 } from '../styles/css'

const style = css(
  {
    position: 'relative',
    maxWidth: 1440,
    minHeight: 'calc(100vh - 160px)',
    marginRight: 'auto',
    marginLeft: 'auto',
    paddingRight: 20,
    paddingLeft: 20,
  },
  media3({
    paddingRight: 80,
    paddingLeft: 80,
  }),
)

type Props = {
}

export default (props: Props) => (
  <section className={style} {...props} />
)
