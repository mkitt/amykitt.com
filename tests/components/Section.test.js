import React from 'react'
import renderer from 'react-test-renderer'
import Section from '../../components/Section'

describe('Section', () => {
  let component

  beforeEach(() => {
    component = (
      <Section />
    )
  })

  it('shows the correct snapshot for the Section component tree', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
