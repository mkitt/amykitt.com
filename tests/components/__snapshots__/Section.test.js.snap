// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section shows the correct snapshot for the Section component tree 1`] = `
<section
  className={
    Object {
      "data-css-1j7662z": "",
    }
  }
/>
`;
